README.md

```diff
@@ -1,7 +1,11 @@
-SeQuant: symbolic tensor algebra in C++
-=======================================
+# SeQuant: Symbolic Tensor Algebra in C++
 
-# Synopsis
+[![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)]()
```

Comment:
wrong licence, we are right now GPL-3 .. I missed this

Reply:
Sorry, will fix that.